sosw/components/test/unit/test_helpers.py

```diff
@@ -810,5 +810,19 @@ def test_is_event_from_sqs__many(self):
                          unwrap_event_recursively(deepcopy(SQS_EVENT_MANY)))
 
 
+    def test_small_int_from_string(self):
+
+        self.assertEqual(small_int_from_string("hello world"), 91)
+        self.assertEqual(small_int_from_string("Hello I am a String without specifying number of digits"), 27)
+        self.assertEqual(small_int_from_string("hello world", 3), 291)
+        self.assertEqual(small_int_from_string("sosw is the best", 1), 0)
+        self.assertEqual(small_int_from_string("sosw is the best", 5), 68280)
+        self.assertEqual(small_int_from_string("sosw is the best", ), 80)
+
+        with self.assertRaises(ValueError):
+            for _ in range(3):
```

Comment:
LOL!
I didn't mean to run checking for Exception 3 times. :)))
Just check once.

Ignore my comment about 3 times. :)